math/src/main/java/com/powsybl/math/matrix/SparseMatrix.java

```diff
@@ -152,21 +152,52 @@ public int getN() {
      * @throws PowsyblException if values are not filled in column order.
      */
     @Override
-    public void setValue(int m, int n, double value) {
-        if (n == currentColumn) {
+    public void setValue(int i, int j, double value) {
+        if (j == currentColumn) {
             // ok, continue to fill row
-        } else if (n > currentColumn) {
+        } else if (j > currentColumn) {
             // start new column
-            columnStart[n] = values.size();
-            currentColumn = n;
+            columnStart[j] = values.size();
+            currentColumn = j;
         } else {
             throw new PowsyblException("Columns have to be filled in the right order");
         }
         values.add(value);
-        rowIndices.add(m);
+        rowIndices.add(i);
         columnStart[columnStart.length - 1] = values.size();
     }
 
+    /**
+     * {@inheritDoc}
+     *
+     * <p>
+     * As sparse matrix is stored in CSC format, values have to be filled in column order.
+     * </p>
+     * @throws PowsyblException if values are not filled in column order.
```

Comment:
Maybe we can be even more specific and say that columns must be filled in ascending order but values inside a column may be filled in any order ?

Reply:
Doc fixed